config/routes.rb

```diff
@@ -3,6 +3,8 @@
   resources :users, only: [:index, :show] do
     resources :posts, only: [:index, :show]
   end
+
+  resources :posts, only: [:index, :show]
```

Comment:
- [ ] **[OPTIONAL]** I see that you are having duplicated routes in your app. I suggest you stick with the original ones, and, in your tests, just create a user before running any test cases and use that id to navigate! 🤓 🔧